src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/PayoutsSection/PayoutsTable/PayoutSplitRow.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  EllipsisVerticalIcon,
+  PencilIcon,
+  TrashIcon,
+} from '@heroicons/react/24/solid'
+import { useWatch } from 'antd/lib/form/Form'
+import { DropdownMenu } from 'components/Navbar/components/DropdownMenu'
+import FormattedNumberInput from 'components/inputs/FormattedNumberInput'
+import { CURRENCY_METADATA } from 'constants/currency'
+import { ONE_BILLION } from 'constants/numbers'
+import round from 'lodash/round'
+import { Split } from 'models/splits'
+import {
+  adjustedSplitPercents,
+  getNewDistributionLimit,
+} from 'utils/v2v3/distributions'
+import { useEditCycleFormContext } from '../../EditCycleFormContext'
+import { PayoutTitle } from './PayoutTitle'
+import { PayoutsTableCell } from './PayoutsTableCell'
+import { PayoutsTableRow } from './PayoutsTableRow'
+
+const Cell = PayoutsTableCell
+
+export function PayoutSplitRow({ payoutSplit }: { payoutSplit: Split }) {
+  const { editCycleForm } = useEditCycleFormContext()
+
+  // useWatch not loading
```

Comment:
If you switch these to `editCycleForm.getFieldValue` everything will load properly, but then when you go to edit the amount of one in the row, everything takes a while to update